gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -191,6 +195,9 @@ public void onDeleteSpec(URI deletedSpecURI, String deletedSpecVersion, Properti
   @Override
   public void onUpdateSpec(Spec updatedSpec) {
     _log.info("Spec changed: " + updatedSpec);
+    if (updatedSpec instanceof FlowSpec) {
```

Comment:
Don't we want to call `onDeleteSpec` and `onAddSpec` for FlowSpec, like we are doing for TopologySpec?

Reply:
I think we don't need delete spec for flow spec? As nothing needed on compiler side when we delete one flow? We just want to make sure the new spec is compilable.